src/renderer/store/modules/invidious.js

```diff
@@ -16,42 +16,48 @@ const getters = {
 }
 
 const actions = {
+  async fetchInvidiousInstancesFromFile({ commit }) {
+    const url = createWebURL('/static/invidious-instances.json')
+
+    const fileData = await (await fetch(url)).json()
+    const instances = fileData.filter(e => {
+      return process.env.SUPPORTS_LOCAL_API || e.cors
+    }).map(e => {
+      return e.url
+    })
+
+    commit('setInvidiousInstancesList', instances)
+  },
+
+  /// fetch invidious instances from site and overwrite static file.
   async fetchInvidiousInstances({ commit }) {
     const requestUrl = 'https://api.invidious.io/instances.json'
-
-    let instances = []
     try {
       const response = await fetchWithTimeout(15_000, requestUrl)
       const json = await response.json()
-      instances = json.filter((instance) => {
+      const instances = json.filter((instance) => {
         return !(instance[0].includes('.onion') ||
           instance[0].includes('.i2p') ||
           !instance[1].api ||
           (!process.env.SUPPORTS_LOCAL_API && !instance[1].cors))
       }).map((instance) => {
         return instance[1].uri.replace(/\/$/, '')
       })
+
+      if (instances.length !== 0) {
+        commit('setInvidiousInstancesList', instances)
+        return true
```

Comment:
Looks like the return values from this function are unused, so they can probably be removed?